nixos/modules/programs/steam.nix

```diff
@@ -6,6 +6,14 @@ let
   cfg = config.programs.steam;
 
   steam = pkgs.steam.override {
+    extraPkgs = pkgs: [
```

Comment:
I feel like this should be done inside of the `fhsenv.nix` file, and not through `extraPkgs`. As this option is used by people to override and add extra packages on their end. So this change would get overwritten when someone does `steam.override { extraPkgs = (pkgs: ...); }` themselves.

Reply:
Well you could do something like

```nix
lib.makeOverridable
  ( { extraPkgs, ... }@args:
    steam.override (args // { extraPkgs = pkgs: [...] ++ extraPkgs pkgs; }) )
  { extraPkgs = pkgs: []; }
```
(by the way, someone more experienced should check if that is the correct way to do it).

But its probably easier to edit `fhsenv.nix` instead..

Reply:
This is still a really hacky way to include a font, and should rather be refactored into something more reproducible inside of the `fhsenv` itself.